Xamarin.Forms.Platform.Android/ViewRenderer.cs

```diff
@@ -92,10 +92,16 @@ protected override void Dispose(bool disposing)
 					_container = null;
 				}
 
-				if (Element != null && _focusChangeHandler != null)
+				if (Element != null)
 				{
-					Element.FocusChangeRequested -= _focusChangeHandler;
-					_focusChangeHandler = null;
+					Element.Behaviors?.Clear();
+					Element.Triggers?.Clear();
```

Comment:
@StephaneDelcroix Could you please write a bit more info, what does "this breaks reuse of Element" means?
This is a serious memory leak, I'm confused why this was closed.